src/edu/ucsb/nceas/metacat/restservice/D1ResourceHandler.java

```diff
@@ -113,7 +114,7 @@
     protected static final String FUNCTION_NAME_UPDATE = "update";
     
     protected ServletContext servletContext;
-    protected static Logger logMetacat;
+    protected static Log logMetacat;
```

Comment:
Use a single log instance for all methods.
```suggestion
    protected static Log logMetacat = LogFactory.getLog(D1ResourceHandler.class);
```